src/PythonQtConversion.cpp

```diff
@@ -947,7 +947,8 @@ double PythonQtConv::PyObjGetDouble(PyObject* val, bool strict, bool &ok) {
 template <typename Map> 
 void PythonQtConv::pythonToMapVariant(PyObject* val, QVariant& result)
 {
-  if (PyMapping_Check(val)) {
+  // check if val is not a string, otherwise AttributeError("'str' object has no attribute 'items'") is raised which is misleading
```

Comment:
I believe this can not just occur for strings, but for other types too. Perhaps it would be better to just clear the error produced by PyMapping_Items instead. (If an error is already set we could skip the whole thing to avoid clearing an external error.)